firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -135,3 +135,77 @@ def test_matching_text(self):
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
                 self.assertIn(match_fragment, (input_text, input_text.upper()))
+
+
+class TestIdentityPopup(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            self.identity_popup.close(force=True)
+        except NoSuchElementException:
+            # TODO: A NoSuchElementException may be thrown here when tests accessing the
+            # identity_popup.popup element are skipped.
+            pass
```

Comment:
Ok, so this is in parity with the Autocomplete popup unit tests. We have to get a bug filed on it. Interestingly `close()` can be called in setUp without this failure. I think we have a wrong window handle here or the focus is somewhere else. @galgeek can you please file this bug?